lbry/blockchain/sync/synchronizer.py

```diff
@@ -259,7 +259,7 @@ async def count_takeovers(self, blocks) -> int:
 
     async def sync_claims(self, blocks) -> bool:
         delete_claims = takeovers = claims_with_changed_supports = claims_with_changed_reposts = 0
-        initial_sync = not await self.db.has_claims()
+        initial_sync = not await self.db.has_filters()
```

Comment:
why this change? i would intuitively expect a function `sync_claims` to check the `claims` table.

Reply:
some tests doesnt create claims, so it will think its the initial sync all the time and try to create indexes that already exist